nservicebus/rabbitmq/connection-settings_tls-details_rabbit_[3,].partial.md

```diff
@@ -0,0 +1,9 @@
+## Transport Layer Security support
+
+In Versions 3.2 and above, the RabbitMQ transport supports creating secure connections to the broker using Transport Layer Security (TLS). For information on how to configure TLS on the RabbitMQ broker, refer to the [RabbitMQ documentation](http://www.rabbitmq.com/ssl.html). To enable TLS support, set the `UseTls` setting to `true` in the connection string. If the RabbitMQ broker has been configured to require client authentication, a client certificate can be specified in the `CertPath` setting. If that certificate requires a password, it can be specified in the `CertPassphrase` setting.
```

Comment:
how about 
instead of

> In Versions 3.2 and above, the RabbitMQ transport supports creating secure connections to the broker using Transport Layer Security (TLS). For information on how to configure TLS on the RabbitMQ broker, refer to the [RabbitMQ documentation](http://www.rabbitmq.com/ssl.html).

this

> In Versions 3.2 and above, secure connections to the broker using [Transport Layer Security (TLS)](http://www.rabbitmq.com/ssl.html) are supported.